CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/Memory.java

```diff
@@ -34,4 +41,58 @@ public void flushMemory(){
         _memory.clear();
         _address = 0;
     }
+
+    public String copyFiles(Args args) throws IOException {
+        File sourcePath = new File(args.get("source_path").toString());
+        File destinationPath = new File(args.get("destination_path").toString());
+        try {
+            copyFolder(sourcePath, destinationPath);
+            return "Copied";
+        } catch (Exception e){
+            return e.getLocalizedMessage().toString();
+        }
+    }
+
+    public static void copyFolder(File src, File dest) throws IOException{
+        if(src.isDirectory()){
+
+            if(dest.exists()) {
+                FileUtils.deleteRecursive(dest);
```

Comment:
definition of function says 'copyFolder', but it looks like copies file, but not folder based on type you pass it as a parameter. Is there a reason to use copyFolder?

Reply:
if you see beyond if else, you can see that it calls copyfolder recursively for each file (Android object) present in src diretory and by file it means both file and folder